lib/samples/display_dimensions/README.md

```diff
@@ -0,0 +1,41 @@
+# Display dimensions
+
+Display dimension features from a mobile map package.
+
+![Image showing the Display Dimensions sample](display_dimensions.png)
+
+## Use case
+
+Dimensions show specific lengths or distances on a map. A dimension may indicate the length of a land parcel, or the distance between two features, such as a fire hydrant and the corner of a building.
+
+## How to use the sample
+
+When the sample loads, it will automatically display the map containing dimension features from the mobile map package. The name of the dimension layer containing the dimension features is displayed in the controls box. Control the visibility of the dimension layer with the "Dimension Layer visibility" switch, and apply a definition expression to show dimensions greater than or equal to 450m in length using the "Definition Expression" switch.
+
+**Note**: the minimum scale range of the sample is set to 1:35000 to maintain readability of the dimension features.
+
+## How it works
+
+1. Create a `MobileMapPackage` specifying the path to the .mmpk file.
+2. Load the mobile map package (mmpk) with `mmpk.load()`.
+3. After the mmpk successfully loads, get the map from the mmpk `map = mmpk.maps.first`.
+4. Loop through the map's layers to find the `DimensionLayer` and get the layer name to display on the UI with `_dimensionLayer.name`.
```

Comment:
```suggestion
4. Loop through the map's layers to find the `DimensionLayer` and get the layer name to display on the UI with `dimensionLayer.name`.
```

Reply:
hmm.. in the code, there is a leading underscore. That's missing from the references in the 5th bullet. Which should I change? Should I use the leading underscore or not?